indi-celestronaux/celestronaux.cpp

```diff
@@ -513,6 +513,17 @@ bool CelestronAUX::updateProperties()
         FirmwareTP[FW_GPS].setText(fwText);
         defineProperty(FirmwareTP);
 
+        // When no HC is attached, the following three commands needs to be send
+	// to the motor controller (MC): MC_SET_POSITION, MC_SET_CORDWRAP_POSITION
+	// and MC_CORDWRAP_ON. These three commands are also send by the HC
+	// to the MC during HC startup and quick align process.
+        // TODO: One can set the HC in pass through mode, that is,
+        // the HC relays the AUX commands only and does not interfere in the communication.
+	if (!m_isHandController && m_MountType == EQ_GEM)
+	{
+	    startupWithoutHC();
+	}
```

Comment:
Why limit to EQ_GEM, we can make it work for all types perhaps in startupWithoutHC can have this?

data[0] = (m_MountType == EQ_GEM) ? 0x40 : 00

wouldn't that be sufficient?